src/main/java/com/scaler/lld/design/assignments/prototype/ConfigurationPrototypeRegistryImpl.java

```diff
@@ -0,0 +1,23 @@
+package com.scaler.lld.design.assignments.prototype;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ConfigurationPrototypeRegistryImpl implements ConfigurationPrototypeRegistry{
```

Comment:
[Minor] Run a formatter/linter through this file. There are some missing spaces.